packages/@uppy/dashboard/src/utils/getDroppedFiles/utils/getFilesAndDirectoriesApi.js

```diff
@@ -0,0 +1,27 @@
+// API implemented in Firefox 42+ and Edge
+module.exports = function getFilesAndDirectoriesApi (input, cb) {
+  const files = []
```

Comment:
I think we should move those util functions that work with drag-drop to `@uppy/utils`, so they can be shared (maybe later) with `drag-drop` plugin? Otherwise it will continue to depend on `drag-drop` module.